src/item_factory.cpp

```diff
@@ -2657,7 +2685,10 @@ void armor_portion_data::deserialize( const JsonObject &jo )
 
     optional( jo, false, "rigid_layer_only", rigid_layer_only, false );
 
-    if( jo.has_array( "encumbrance" ) ) {
+    if( jo.has_array( "encumberance_modifiers" ) ) {
```

Comment:
You'll need to do this everywhere else too.
```suggestion
    if( jo.has_array( "encumbrance_modifiers" ) ) {
```